src/components/permissions/routes.ts

```diff
@@ -116,11 +119,23 @@ const routeVerification = async (request: any, reply: any) => {
   }
 }
 
+// Serve prefs to front-end
+const routeGetPrefs = async (request: any, reply: any) => {
+  const prefs = JSON.parse(
+    readFileSync(path.join(getAppEntryPointDir(), '../preferences.json'), 'utf8')
+  )
+  const languageOptions = JSON.parse(
+    readFileSync(path.join(getAppEntryPointDir(), '../localisation/languages.json'), 'utf8')
+  )
+  reply.send({ preferences: prefs.web, languageOptions })
```

Comment:
Hmm I see so this would get passed in the first iteration between server and front-end app?

Reply:
Yeah, so the front-end starts off by loading prefs and languageOptions before it does anything else.

It already did that for languageOptions, I've just updated it so it gets prefs at the same time.

Reply:
Cool